guides/customization/tailscale.md

```diff
@@ -200,11 +196,12 @@ ENV http_proxy=http://localhost:3128
 
 ## Step 2: Authenticate to Tailscale
 
-Start a workspace using the container image. Initially, `tailscaled` should be
-running, but will indicate that it requires authentication:
+[Create a workspace] using the container image. Initially, `tailscaled` should
+be running, but it will indicate that it requires authentication:
 
 ```console
-$ systemctl status tailscaled
+systemctl status tailscaled
```

Comment:
I think the `$` prefix is important to distinguish that: (1) this command is not run as root; and (2) to distinguish the prompt from the example output

but if our convention is to skip the example output or put the output in a separate fenced code block, we can do that